source/clear-linux/tutorials/dlaas.rst

```diff
@@ -0,0 +1,240 @@
+
+Kubernetes setup
+****************
+
+Follow the instructions in the :ref:`kubernetes` tutorial to get set up on
+|CL|. The kubernetes community also has
+`instructions for creating a cluster`_.
+
+Kubernetes networking
+*********************
+
+We used `flannel`_ as the network provider for these tests. If you are
+comfortable with another network layer, refer to the Kubernetes
+`networking documentation`_ for setup.
+
+Images
+******
+
+We need to add `launcher.py` to our docker image to include the Intel® Deep
+Learning Stack and put the benchmarks repo in the correct location. From the
+docker image, run the following:
+
+.. code-block:: bash
+
+   mkdir -p /opt
+   git clone https://github.com/tensorflow/benchmarks.git /opt/tf-benchmarks
+   cp launcher.py /opt
+   chmod u+x /opt/*
+
+Your entry point now becomes "/opt/launcher.py".
+
+This will build an image which can be consumed directly by TFJob from
+kubeflow. We are working to create these images as part of our release
+cycle.
+
+ksonnet\*
+*********
+
+Kubeflow uses ksonnet* to manage deployments, so we need to install that before setting up Kubeflow. On |CL|, follow these steps:
+
+.. code-block:: bash
+
+   swupd bundle-add go-basic-dev
+   export GOPATH=$HOME/go
+   export PATH=$PATH:$GOPATH/bin
+   go get github.com/ksonnet/ksonnet
+   cd $GOPATH/src/github.com/ksonnet/ksonnet
+   make install
+
+After the ksonnet installation is complete, ensure that binary `ks` is
+accessible across the environment.
+
+Kubeflow
+********
+
+Once you have Kubernetes running on your nodes, you can setup `Kubeflow`_ by
+following these instructions from their `quick start guide`_.
+
+.. code-block:: bash
+
+   export KUBEFLOW_SRC=$HOME/kflow
+   export KUBEFLOW_TAG=”v0.3.2”
+   export KFAPP=”kflow_app”
+   export K8S_NAMESPACE=”kubeflow”
+
+   mkdir ${KUBEFLOW_SRC}
+   cd ${KUBEFLOW_SRC}
+   ks init ${KFAPP}
+   cd ${KFAPP}
+   ks registry add kubeflow github.com/kubeflow/kubeflow/tree/${KUBEFLOW_TAG}/kubeflow
+   ks pkg install kubeflow/core
+
+Now you have all the required kubeflow packages, and you can deploy the primary one for our purposes: tf-job-operator.
+
+.. code-block:: bash
+
+   ks env rm default
+   kubectl create namespace ${K8S_NAMESPACE}
+   ks env add default --namespace "${K8S_NAMESPACE}"
+   ks generate tf-job-operator tf-job-operator
+   ks apply default -c tf-job-operator
+
+This creates the CustomResourceDefinition(CRD) endpoint to launch a TFJob.
+
+Running an Intel Deep Learning as a Service Stack TFJob
+=======================================================
+
+#. Select this link for the `DLaaS ksonnet registries for deploying TFJobs`_.
+
+   #. Install DLaaS TFJob componets as follows:
+
+      .. code-block:: bash
+
+         ks registry add dlaas-tfjob github.com/clearlinux/dockerfiles/tree/master/stacks/dlaas/kubeflow/dlaas-tfjob
+
+         ks pkg install dlaas-tfjob/dlaas-bench
+
+#. Download and copy these files into:
```

Comment:
@mvincerx Lines 197-201 are not needed anymore, the previous command does the work.

Reply:
Done.